scss/config/_fonts.scss

```diff
@@ -36,10 +38,8 @@ $sans-font: (
 /// A readable monospace font for code previews.
 ///
 /// @font 'code' (regular, bold)
-///   <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700" rel="stylesheet">
+///   <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400,600" rel="stylesheet">
```

Comment:
Since we changed this to 600, but are still calling "bold" the browser associates that to 700. So are we loading a faux bold here in some browsers? I wonder if we should use 600 instead when we call it later?

Reply:
ok, that makes sense. @stacyk do you want to make that change, so we can get this merged?

Reply:
If I'm not mistaken, addressing this comment is the only thing remaining on this PR before merge?